amp-pwa-reader/src/js/Article.js

```diff
@@ -16,20 +16,21 @@
 
 class Article {
 
-  constructor(url, card) {
+  constructor(url, card, streaming) {
     this.url = shadowReader.backend.getAMPUrl(url);
+    this.proxyUrl = this.urlProxy(this.url);
```

Comment:
why proxyUrl and not url? The fact that it is a proxy under the hood is a detail not important to the client, no?

Reply:
There's code in `Article.js` that uses the original Guardian URL.  This is useful when the article gets stuck in a shadow root; it makes sense for `AMP.attachShadowDoc()` and `AMP.attachShadowDocAsStream()`.  On the other hand, when `fetch()` retrieves the article, it retrieves it from a proxy server, not from the original Guardian.

Agreed that the client shouldn't care that there's a proxy.  What's a better name for this, then?  `fetchUrl`?  It's only used during fetching and streaming.

Reply:
mhh can you show me an example of the difference of this.url and this.proxyUrl? how are they different?

Reply:
So, when`this.url` is `"https://amp.theguardian.com/fashion/2018/may/29/balenciaga-shirt-t-shirt-cost-ikea-bag-tote-crocs"`,
`this.proxyUrl` could be `"https://seed-octagon.glitch.me/https%3A%2F%2Famp.theguardian.com%2Ffashion%2F2018%2Fmay%2F29%2Fbalenciaga-shirt-t-shirt-cost-ikea-bag-tote-crocs"`

Reply:
Looking at this again - I think we killed this in the SSR PR anyway 🙃